hms-app/src/components/hackathon-details/NewCategory.tsx

```diff
@@ -127,7 +96,7 @@ function NewCategory() {
           />
         </Card.Section>
         <Group position="right" mt="xl">
-          <Button disabled={!categoryText.titleSet} onClick={submitForm}>
+          <Button disabled={!hasTitle} onClick={submitForm}>
```

Comment:
so this bit I would advise to write a function that looks at the current state and determines if this should be disabled. You could do away with the `hasTitle` state hook and just do something like:

```
function submitIsEnabled(): boolean {
  return !!categoryText.title
}
```